cli/flox-rust-sdk/src/data/mod.rs

```diff
@@ -1,12 +1,9 @@
-mod canonical_path;
 mod flox_version;
-mod version;
 
 use std::fmt::Display;
 
-pub use canonical_path::{CanonicalPath, CanonicalizeError};
+pub use flox_core::canonical_path::{CanonicalPath, CanonicalizeError};
```

Comment:
Why are we re-exporting this from another crate?

Reply:
Because when I first added `flox-core` I was trying to minimize the diff. We can now use `flox_core` directly throughout `flox-rust-sdk` if we want